src/types/appleSk2.ts

```diff
@@ -96,6 +96,7 @@ export type TransactionSk2 = {
   deviceVerification: string;
   deviceVerificationNonce: string;
   expirationDate: number;
+  environment: 'Production' | 'Sandbox' | 'Xcode' | null;
```

Comment:
Why null and not undefined/ optional. I think we are optional fields everywhere else

Reply:
Can you please add a documentation comment saying that it's only available on iOS 16?

Thank you for your contribution

Reply:
I don't think it's only available in iOS 16, coz the swift implementation in serialize helper in my PR is using either `transaction.environment` (which is stable since iOS 16) or `transaction.environmentStringRepresentation` (which is available since iOS 15 but in some rare cases it could return not an expected environment string according to docs in StoreKit from XCode - see attachments for details)